src/controller/CHIPDeviceController.cpp

```diff
@@ -684,6 +684,14 @@ CHIP_ERROR DeviceCommissioner::Init(CommissionerInitParams params)
 #endif
 
     mPairingDelegate = params.pairingDelegate;
+    if (params.defaultCommissioner != nullptr)
+    {
+        mDefaultCommissioner = params.defaultCommissioner;
+    }
+    else
+    {
+        mDefaultCommissioner = &mAutoCommissioner;
```

Comment:
Why still keep an mAutoCommissioner? Why not update the samples to construct an AutoCommissioner and pass it in? Then we wouldn't have to call this `mDefaultCommissioner` but rather `mComissionerDriver` or such

Reply:
That's kind of the direction I'm going, but I just haven't gotten there yet - can this be addressed in a follow up? It ends up being a bit dodgy with the CI.

Reply:
Sure